claims-data/service/src/main/java/uk/gov/justice/laa/dstew/payments/claimsdata/converter/BulkSubmissionCsvConverter.java

```diff
@@ -86,9 +86,14 @@ public CsvSubmission convert(MultipartFile file) {
             csvSchedule =
                 objectMapper.convertValue(csvBulkSubmissionRow.values(), CsvSchedule.class);
           }
-          case CsvHeader.OUTCOME ->
-              csvOutcomes.add(
-                  objectMapper.convertValue(csvBulkSubmissionRow.values(), CsvOutcome.class));
+          case CsvHeader.OUTCOME -> {
+            var submissionRowValues = csvBulkSubmissionRow.values();
+            if (StringUtils.isEmpty(submissionRowValues.get("matterType"))) {
+              throw new BulkSubmissionFileReadException("Matter type missing in outcome data");
+            }
+            csvOutcomes.add(
+                objectMapper.convertValue(csvBulkSubmissionRow.values(), CsvOutcome.class));
+          }
```

Comment:
Can we something like this 
submissionRowValues.computeIfAbsent("matterType", key-> {
              throw new BulkSubmissionFileReadException("Matter type missing in outcome data");
            });